.github/workflows/test.yml.orig

```diff
@@ -0,0 +1,668 @@
+name: ovn-ci
```

Comment:
this is a file with git conflicts ... 

Did you mean for this to be uploaded this way ? ...

Furthermore, could you please isolate CI configurations in a separate commit ?

Reply:
Removing...